projects/angular/src/wizard/wizard.ts

```diff
@@ -471,7 +471,15 @@ export class ClrWizard implements OnDestroy, AfterContentInit, DoCheck {
   }
 
   private listenForPageChanges(): Subscription {
-    return this.navService.currentPageChanged.subscribe(() => this.currentPageChanged.emit());
+    return this.navService.currentPageChanged.subscribe(() => {
```

Comment:
When we return these subscriptions, where do we unsubscribe them?

Reply:
Subscriptions are unsubscribed in the [ngOnDestroy on line 269](https://github.com/vmware/clarity/pull/6670/files#diff-76309545a2b4ee57076610f8514507e223a4ccf73b9fe8be31ff8b6cd81c4ebfR269)